server/bundler/bin/bundler.js

```diff
@@ -114,7 +114,7 @@ webpack( webpackConfig, function( error, stats ) {
 	} ).map( function( chunk ) {
 		return path.join( process.cwd(), 'public', chunk.file );
 	} );
-	files.push( path.join( process.cwd(), 'public', 'vendor.js' ) );
-
+	files.unshift( path.join( process.cwd(), 'public', 'vendor.js' ) );
```

Comment:
this may not have been the best place to test out the resolve conflicts feature of GitHub.  Is there anything different (or worse) about this method?